cmd/main.go

```diff
@@ -89,7 +84,11 @@ func main() {
 		configuration.SetInterfaces(ifaces)
 	}
 
-	demo := democem.NewDemoCem(configuration, *remoteSki)
+	demo := democem.NewDemoCem(configuration)
+
+	if len(os.Args) > 1 && *remoteSki != "" {
```

Comment:
We shouldn't check of the amount of args here, but rather:

```
if remoteSki != nil && *remoteSki != "" {
```

If the nil check above stays, this could also be removed here